documentation/roadmap.md

```diff
@@ -0,0 +1,85 @@
+# Roadmap
+
+The goal of this roadmap is to give you a big picture about what is coming for
+Kotools Types.
+
+Here’s a list of the most important things we focus on delivering:
+
+- **Aligning type converters with our design goals on error handling** by
+  suffixing them with `OrNull` if they can fail, or by not suffixing them if
+  they shouldn't fail, for being agnostic about how users handle errors.
+- **Clearing separation of the experimental API from the stable API** by
+  centralizing experimental declarations in the `kotools.types.experimental`
+  package.
+- **Adhering back to [Semantic Versioning][semantic-versioning]** for
+  communicating clearly what's changed in the stable API.
+
+## Version 4.3.2 <a id="v4.3.2"></a>
+
+- Hiding internals from Java.
```

Comment:
Hiding internals from Java can break the code of Java users: it should be done in the next major release instead.